src/Model/State.re

```diff
@@ -125,6 +125,21 @@ let defaultKeyBindings =
         ~command=Commands.List.select.id,
         ~condition="listFocus" |> WhenExpr.parse,
       ),
+      bind(
+        ~key="<C-v>",
+        ~command=Commands.List.selectVertical.id,
+        ~condition="listFocus" |> WhenExpr.parse,
+      ),
+      bind(
+        ~key="<C-s>",
+        ~command=Commands.List.selectHorizontal.id,
+        ~condition="listFocus" |> WhenExpr.parse,
+      ),
```

Comment:
This also conflicts with the `save` keybinding, but I think this one is OK - understandable that save wouldn't occur when the quickmenu is up.